README.rst

```diff
@@ -157,6 +164,38 @@ This hook could be useful for directly modifying the response content, for examp
   def add_custom_header(page, response):
       response['Wagtail-Is-Shared'] = '1'
 
+Mixins
+------
+
+``ShareableRoutablePageMixin``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+ .. |RoutablePageMixin| replace:: ``RoutablePageMixin``
+ .. _RoutablePageMixin: https://docs.wagtail.io/en/stable/reference/contrib/routablepage.html
+
+By default, Wagtail Sharing cannot share draft content on routable pages (pages that use |RoutablePageMixin|_ to define mutliple sub-URLs). To ensure that draft content fields on routable pages are sharable, Wagtail Sharing provides an subclass of |RoutablePageMixin|_, ``ShareableRoutablePageMixin``. 
```

Comment:
Minor typo here: "mutliple".

More generally, the use of "by default" here might be a little confusing at first. It might be clearer to say something like "Wagtail's RoutablePageMixin is not compatible with W-S, instead you need to use ShareableRoutablePageMixin".

Reply:
Used the recommended language here.